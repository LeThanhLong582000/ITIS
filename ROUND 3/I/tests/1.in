17
3 5 6 11 13 15 16 17 20 23 26 30 33 38 40 41 47
499977102434654931 907855997051853974
