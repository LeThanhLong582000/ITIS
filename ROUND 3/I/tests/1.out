264380052455249932
